build.sh

```diff
@@ -256,6 +258,19 @@ function build_desktop {
     fi
 }
 
+function build_desktop_tools_for_ios {
+    # Supress intermediate desktop tools install and Java
```

Comment:
nit^99: suppress

Reply:
Yes I noticed it too, and decided to leave it as it is